examples/mock.rs

```diff
@@ -46,7 +46,11 @@ async fn make_watch_only_events(client: &Client) -> Result<u64> {
 #[tokio::main]
 async fn main() {
     let mock = test::Mock::new();
-    let client = Client::default().with_url(mock.url());
+    let client = Client::default()
+        .with_url(mock.url())
+        // disabled schema validation is required for mocks to work;
```

Comment:
The problem here is that if you have some actor that uses internally `Client` with enabled validation, you cannot write a test using mock anymore =(